### DISCUSSION TITLE
Dynamic DNS for rounter

### DISCUSSION THREAD

**Discussion by craigpress** [chadgeary/cloudblock#22] [Q&A]
Is there a way to set things up so that if your router has a dynamic IP we can point the firewall to a dynamic DNS address? I am worried about the process of updating the firewall settings for the new IP.

**Comment by chadgeary** [comment]
Hey @craigpress thats a great question.

You're not the first to ask - it's come up in discord too.

I really want to provide a method for updating the firewall rules quickly - and there are some good ideas floating around. Until something is written, I suggest sticking with Wireguard on your most used clients.

Which cloud provider are you using?

**Comment by craigpress** [comment]
I picked oracle just because the 10tb data allowance.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
